model/browser/events.yaml

```diff
@@ -0,0 +1,104 @@
+groups:
+  - id: event.browser.resource_timing
+    stability: development
+    type: event
+    name: browser.resource_timing
+    brief: >
+      This event describes the timing metrics as provided by PerformanceResourceTiming Performance API.
+      These metrics are related to fetching a resource, such as
+      XMLHttpRequest, Fetch, sendBeacon APIs, SVG, image or script.
+    note: >
+      This event captures data from the [ResourceTiming](https://www.w3.org/TR/resource-timing/).
+      It is recommended to be captured by using the PerformanceResourceTiming API.
+      If the page starts unloading before the page loads,
+      then the partial data can be captured by calling the [performance.getEntriesByType](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByType) method.
+
+    body:
+      id: browser.resource_timing
+      requirement_level: required
+      stability: development
+      type: map
+      fields:
+        - id: name
```

Comment:
I don't think `name` is the right thing for a url here. Maybe just `url` would be more concise?

Reply:
Yeah, now that `url.full` is stable maybe we should consider lifting this to a top level attribute, where from an instrumentation perspective it would include the full redacted query string -- although for this event you "generally" don't need the full url, just the scheme, domain and path -- but splitting out to 3 different attributes doesn't really fit the need of this event....

Lets change this to `url` and add wording that it explicitly doesn't include the query string or any leading credentials (so it's formatted as `<scheme>:\\<full domain>\<path>` only
What I mean by this is that
`https://username:password@www.example.com/path?a=b&c=d` would be sent as `https://www.example.com/path` basically both removing any potential PII and reducing the cardinality of the name.

Reply:
and change it to required

Reply:
Make sure the Note is present:  it explicitly doesn't include the query string or any leading credentials (so it's formatted as <scheme>:\\<full domain>\<path> only https://username:password@www.example.com/path?a=b&c=d would be sent as https://www.example.com/path basically both removing any potential PII and reducing the cardinality of the name. (auto-reconstruct url instead of actual full url)